OpenDreamShared/Net/Packets/PacketDeltaGameState.cs

```diff
@@ -272,6 +275,11 @@ private enum ClientValueID {
                     stream.WriteByte((byte)ClientDelta.NewPerspective.Value);
                 }
 
+                if (newSeeInvisible)
+                {
+                    stream.WriteByte((byte)ClientDelta.NewSeeInvisible.Value);
```

Comment:
You have to write which value this is before actually writing the value
```suggestion
                    stream.WriteByte((byte)ClientValueID.SeeInvisible);
                    stream.WriteByte((byte)ClientDelta.NewSeeInvisible.Value);
```